src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+
+Starting with v5, any permissions not explicitly declared in the manifest are by default not granted. 
+
+### Network
+You have to specify unlock the `network` module in `requiredPermissions` by defining domains from which network requests are allowed. `domains` accepts an array of strings or a singular string `"all"`.
+```json
+{
+    "requiredPermissions": {
+        "network": {
+            "domains": [
+                "https://source.unsplash.com",         
+            ]
+        }
+    }
+}
+```
+```html
+<img src='https://source.unsplash.com/random'>
+```
+### Clipboard
+To grant read and/or write access to the system clipboard, declare it in `requiredPermissions`.
+`clipboard` accepts:
+* `readAndWrite` for read/write access
+* `read` for read-only access. 
+
+```json
+{
+    "requiredPermissions": {
+        "clipboard": "readAndWrite"
+    }
+}
+```
+ 
+```js
+const clipboard = navigator.clipboard;
+const dataTransferProviders = {
+    'text/plain': 'Sample text'
+};
+clipboard.writeText(dataTransferProviders).then(
+    (result) => {...},
+    (error) => {...}
+);
+```
+
+### Local Filesystem
+`localFileSystem` accepts:
+* `request`: Allows the plugin to access the local file system using token provided by UXP.
+* `plugin`: Allows the plugin to access the plugin's sandbox, but doesn't enable file picker APIs.
+  
+```json
+{
+    "requiredPermissions": {
+        "localFileSystem": "request"
+    }
+}
+```
+ 
+```js
+const fs = require('uxp').storage.localFileSystem;
+const file = await fs.getFileForSaving('manifest_demo.txt');
+await file.write('Manifest v5 demo');
+```
+If `plugin` is not specified for plugin storage, the manifest will default to that setting.
+
+### Launch Process
+The `launchProcess` permission in the manifest controls the ability to launch applications and open files in other applications. 
+
+Before manifest v5, `shell.openPath()` was enabled using the feature flag `kEnableShellOpenPath`. Additionally, `openExternal()` was only accessible with the following code block in manifest: 
+```json
+"permissions": {        
+    "launchProcess": 
+        "request" | "any"    
+}
+```
+
+With manifest v5, the feature flag has been removed by v5 and the `launchProcess` permission must be declared to use `openExternal()` or `openPath()`. 
+```json
+"permissions": {        
+    "launchProcess": {    
+        // allows launching files with specified URI schemes         
+        "schemes": 
+            [ "https", "slack", "adbxd" ],   
+        // allows opening files with the specified file extensions          
+        "extensions": 
+            [ ".pdf", ".xd", ".ps" ],        
+    },   
+}
+```
+Both openPath() and openExternal() rely on this permission set, and upon either function call, the user will get a runtime consent dialog. Only after they agree will the API call execute. 
```

Comment:
Ticks around `openPath` and `openExternal`. Don't use parens here.

Reply:
Can you explain why no parens here?

Reply:
That's an interesting question:

With parens, we could _explicitly_ refer to _calling_ the function `openPath`. I.e., `openPath()` would refer to a call of the `openPath` function. But this, again, leaves us with two interesting questions:

1. do we want to make that distinction (we should only make it if we really need it, as it could be confusing to beginners)
2. if we do want this distinction -- which one is it in this case? Does the function call or the function itself rely on the permission? That question is actually more complicated than it might seem at first 😅